src/testUtils/testDefaults.ts

```diff
@@ -37,30 +39,30 @@ export const DEFAULT_REQUEST_ID = 'ARequestID'
 export const INVALID_ENTITY_ID = 'invalid/entity/id' as unknown as number
 
 export const DEFAULT_DUE_DATE = {
-    recurring: false,
+    isRecurring: false,
     string: 'a date string',
     date: DEFAULT_DATE,
 }
 
 export const INVALID_DUE_DATE = {
     ...DEFAULT_DUE_DATE,
-    recurring: 'false',
+    isRecurring: 'false',
 }
 
 export const DEFAULT_QUICK_ADD_RESPONSE: QuickAddTaskResponse = {
-    id: DEFAULT_TASK_ID,
-    projectId: DEFAULT_PROJECT_ID,
+    id: parseInt(DEFAULT_TASK_ID),
```

Comment:
Why do we convert these properties?

https://developer.todoist.com/sync/v9/#quick-add-an-item

Reply:
Ah, I was basing this still on v8 🤦🏻‍♂️ I'll update this PR to reflect the v9 changes.